osgp/platform/osgp-adapter-ws-core/src/main/java/org/opensmartgridplatform/adapter/ws/core/application/config/ResponseUrlDataCleanupScheduledJobConfig.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright 2022 Alliander N.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+package org.opensmartgridplatform.adapter.ws.core.application.config;
+
+import javax.annotation.PostConstruct;
+import org.opensmartgridplatform.adapter.ws.core.application.services.ResponseUrlDataCleanupJob;
+import org.opensmartgridplatform.shared.application.scheduling.OsgpScheduler;
+import org.quartz.SchedulerException;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.PropertySource;
+
+@Configuration
+@PropertySource("classpath:osgp-adapter-ws-core.properties")
+@PropertySource(value = "file:${osgp/Global/config}", ignoreResourceNotFound = true)
+@PropertySource(value = "file:${osgp/AdapterWsCore/config}", ignoreResourceNotFound = true)
+public class ResponseUrlDataCleanupScheduledJobConfig {
```

Comment:
Would it make sense to combine both cleanup jobs into one single config/job? (ResponseData and ResponseUrlData are related and they both seem to use the same cronExpression and retention time)

Reply:
Since they both seem to use the same properties, have you considered configuring the 2 jobs in 1 config? 
Something like this:
`  @PostConstruct
  private void initializeScheduledJobs() throws SchedulerException {
    this.osgpScheduler.createAndScheduleJob(
        ResponseDataCleanupJob.class, this.cronExpression);
    this.osgpScheduler.createAndScheduleJob(
        ResponseUrlDataCleanupJob.class, this.cronExpression);
  }`
Otherwise I guess I would have expected different property names for both jobs.